### PR TITLE
directory restructure and fixed dependency

### PR DESCRIPTION
**Pull request by sanjayuconn**
# Peer Review Information

- Directory restructure
- Fixed dependencies and linting

Doc: https://docs.google.com/document/d/1JVgp-68UyqaLjmMLbLmSFtj34pjAfYUDUx1b1RQs_gE/edit#

# Pull Request Check List

- [ ] Added **tests** for changed code.
- [ ] Updated **documentation** for changed code.

### CODE REVIEW

**Comment by BlairLee** [comment]
It seems you only change dataset, stats related imports. Do you want to change other imports as described in this [doc](https://docs.google.com/document/d/1JVgp-68UyqaLjmMLbLmSFtj34pjAfYUDUx1b1RQs_gE/edit#heading=h.v7t1ynb2u4tr)?

File: datasetinsights/datasets/simulation/__init__.py [hunk 1]
```diff
@@ -1,6 +1,14 @@
+# flake8: noqa
```
Comment (adason) [inline datasetinsights/datasets/simulation/__init__.py:1]: Why ignore all falke8 error here?

File: tests/evaluation_metrics/conftest.py [hunk 2]
```diff
@@ -1,10 +1,12 @@
+"""conftest."""
 import pytest
 
 from datasetinsights.data.bbox import BBox2D
 
 
 @pytest.fixture
 def get_gt_pred_bbox():
+    """get gt pred bbox."""
```
Comment (adason) [inline tests/evaluation_metrics/conftest.py:9]: Python docstrings for unittests are usually nice to have. Why go through every tests and adding docstings does not add helpful info (as this is simply the test function name)?

File: datasetinsights/datasets/protos/string_int_label_map_pb2.py [hunk 3]
```diff
@@ -0,0 +1,185 @@
+# -*- coding: utf-8 -*-
```
Comment (adason) [inline datasetinsights/datasets/protos/string_int_label_map_pb2.py:1]: Why is this file modified? This is a file that was generated by protobuf and it should never be modified manually.

**Comment by adason** [comment]
> It seems you only change dataset, stats related imports. Do you want to change other imports as described in this [doc](https://docs.google.com/document/d/1JVgp-68UyqaLjmMLbLmSFtj34pjAfYUDUx1b1RQs_gE/edit#heading=h.v7t1ynb2u4tr)?

Agree with @BlairLee. I understand "import refactor" might be multiple steps. Can you make short comments on what will be included in this review and what will be addressed in the followup reviews?

File: datasetinsights/datasets/protos/string_int_label_map_pb2.py (hunk 3 above)
Comment (sanjayuconn) [reply datasetinsights/datasets/protos/string_int_label_map_pb2.py:1]: Its been modified by black. linting fix. After restructuring its assuming it a new file.

$ black .
reformatted /Users/sanjay.vishwakarma/Desktop/projects/thea_dir_change/datasetinsights/datasets/protos/string_int_label_map_pb2.py

File: tests/evaluation_metrics/conftest.py (hunk 2 above)
Comment (sanjayuconn) [reply tests/evaluation_metrics/conftest.py:9]: Its pep257 linting requirement

File: datasetinsights/datasets/simulation/__init__.py (hunk 1 above)
Comment (sanjayuconn) [reply datasetinsights/datasets/simulation/__init__.py:1]: init I have only defined import but not used so flak8 complains.

**Comment by sanjayuconn** [comment]
> It seems you only change dataset, stats related imports. Do you want to change other imports as described in this [doc](https://docs.google.com/document/d/1JVgp-68UyqaLjmMLbLmSFtj34pjAfYUDUx1b1RQs_gE/edit#heading=h.v7t1ynb2u4tr)?



This commit only include directory restructure and make sure all functionalities are in place. Btw, import structure we designed for the end user. In our project internally we will use full import.
